src/values/indexables/indexable-builder-versions.php

```diff
@@ -18,13 +18,13 @@ class Indexable_Builder_Versions {
 	 */
 	protected $indexable_builder_versions_by_type = [
 		'date-archive'      => self::DEFAULT_INDEXABLE_BUILDER_VERSION,
```

Comment:
I'm probably missing something here, but why don't we up the builder version for dates? 

My thinking is that since we add the `is_publicly_viewable` property, we should up this too, but maybe I don't get it :)

Reply:
You're right! I missed that one. Thanks for noticing!